google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/apis/CloudLibraryProjectState.java

```diff
@@ -88,11 +88,11 @@ public void projectsScheduled() {
 
   /** Returns the set of {@link CloudLibrary} currently configured on the given {@link Module}. */
   Set<CloudLibrary> getCloudLibraries(Module module) {
-    return moduleLibraryMap.get(module);
+    return moduleLibraryMap.getOrDefault(module, ImmutableSet.of());
```

Comment:
this are improvements unrelated to this PR. This has the potential of NPE'ing if this method is called before the Maven sync occurs (or if its called with an invalid module).